servicetalk-opentelemetry-http/src/main/java/io/servicetalk/opentelemetry/http/OpenTelemetryHttpServerFilter.java

```diff
@@ -41,20 +39,26 @@
 import io.opentelemetry.instrumentation.api.semconv.http.HttpServerMetrics;
 import io.opentelemetry.instrumentation.api.semconv.http.HttpSpanNameExtractor;
 
-import java.util.function.UnaryOperator;
-
 /**
  * A {@link StreamingHttpService} that supports
  * <a href="https://opentelemetry.io/docs/instrumentation/java/">open telemetry</a>.
  * <p>
  * The filter gets a {@link Tracer} with {@value #INSTRUMENTATION_SCOPE_NAME} instrumentation scope name.
  * <p>
- * Append this filter before others that are expected to see {@link Scope} for this request/response. Filters
- * appended after this filter that use operators with the <strong>after*</strong> prefix on
- * {@link StreamingHttpService#handle(HttpServiceContext, StreamingHttpRequest, StreamingHttpResponseFactory)
- * response meta data} or the {@link StreamingHttpResponse#transformMessageBody(UnaryOperator) response message body}
- * (e.g. {@link Publisher#afterFinally(Runnable)}) will execute after this filter invokes {@link Scope#close()} and
- * therefore will not see the {@link Span} for the current request/response.
+ * This filter propagates the OpenTelemetry {@link Context} so the ordering of filters is crucial.
+ * <ul>
+ *     <li>Append this filter before others that are expected to see {@link Context} for this request/response.</li>
+ *     <li>If auto-draining of the request body is desired, add the
+ *     {@link io.servicetalk.http.utils.HttpRequestAutoDrainingServiceFilter} immediately after.</li>
```

Comment:
Should it be something like: "If you'd like to see active Span for auto-drained requests (when request payload was ignored in the service), add the..."?
Bcz if they don't (most ppl don't), they may not have an active span in `onExchangeFinally()` callback